share/container-build.sh

```diff
@@ -2,17 +2,13 @@
 
 #
 # SPDX-FileCopyrightText:  2023, Iker Pedrosa <ipedrosa@redhat.com>
+# SPDX-FileCopyrightText:  2024, Iker Pedrosa <ipedrosa@redhat.com>
 #
 # SPDX-License-Identifier:  BSD-3-Clause
 #
 
-for FILE in share/containers/*; do
-	IFS='/'
-	read -ra ADDR <<< "$FILE"
-	IFS='.'
-	read -ra ADDR <<< "${ADDR[2]}"
-	IFS=''
-	if ! docker build -f $FILE . --output build-out/${ADDR[0]}; then
-		exit
-	fi
-done
+pushd share/ansible/ || exit
+ansible-playbook playbook.yml -i inventory.ini -e 'distribution=alpine'
+ansible-playbook playbook.yml -i inventory.ini -e 'distribution=debian'
+ansible-playbook playbook.yml -i inventory.ini -e 'distribution=fedora'
+popd || exit
```

Comment:
Actually, being the last line of the script, we don't need to popd(1), since it doesn't affect the caller.

```sh
alx@debian:~$ sh -c 'cd /tmp'
alx@debian:~$ 
```

Reply:
Right. I've removed it.